test/linalg/runtests.jl

```diff
@@ -0,0 +1,31 @@
+module TestLinAlg
+
+using LightGraphs
+using LightGraphs.SimpleGraphs
+using LightGraphs.LinAlg
+
+using Base.Test
+
+testdir = dirname(@__FILE__)
+
+testgraphs(g) = [g, Graph{UInt8}(g), Graph{Int16}(g)]
+testdigraphs(g) = [g, DiGraph{UInt8}(g), DiGraph{Int16}(g)]
+
+# some operations will create a large graph from two smaller graphs. We
+# might error out on very small eltypes.
+testlargegraphs(g) = [g, Graph{UInt16}(g), Graph{Int32}(g)]
+testlargedigraphs(g) = [g, DiGraph{UInt16}(g), DiGraph{Int32}(g)]
+
+tests = [
+    "graphmatrices",
+    "spectral"
+]
+
+@testset "LightGraphs.LinAlg" begin
```

Comment:
I haven't seen testsets in a module before. Is there an example of this in Base or in the wild?

I would make a function in `TestLinAlg` called `test_linalg()` or something and put this block in there, then call that function from outside the module in this file. Does that make sense?

Same think with the SimpleGraph tests.

Reply:
I haven't either, actually. I'll expose a `test_linalg()`.

Reply:
Actually, I'm just collapsing this by getting rid of the module entirely.